packages/react-instantsearch/src/core/createInstantSearchManager.error.test.js

```diff
@@ -51,21 +48,26 @@ describe('createInstantSearchManager', () => {
 
         expect(ism.store.getState().error).toBe(null);
 
-        jest.runAllTimers();
+        return Promise.resolve().then(() => {
+          jest.runAllTimers();
```

Comment:
Is it really necessary to have both? Is it because of the use of both timeouts and promises?